war/src/main/webapp/help/system-config/master-slave/availability_zh_CN.html

```diff
@@ -0,0 +1,45 @@
+<div>
+    决定Jenkins的启动和停止。
+
+    <dl>
+        <dt><b>
+            尽可能保持代理在线
+        </b></dt>
+        <dd>
+            该模式下，Jenkins会尽可能让代理保持在线。
+            <p>
+            如果该代理由于临时性网络故障，Jenkins会定期尝试重启它。
+        </dd>
+
+        <dt><b>
+            让代理在特定的时间段内在线或者离线
+        </b></dt>
+        <dd>
+            该模式下，Jenkins会根据一个计划表来启动代理，并保持指定的时长。
+            <p>
+            如果在计划周期内代理掉线，Jenkins会定期尝试重启它。
+            <p>
+            当代理在线时间达到字段<i>计划启动的时间</i>，它将会被下线。
```

Comment:
I'm only spot checking using Google translate, so I'm likely wrong, but this line seems wrong to me.